### PR TITLE
add Error callback to transcribe Method

### PR DESCRIPTION
**Pull request by drummer3333**
If errors occur while processing the data or the connection has problems, they where logged but not available the the API. I added Consumer for all occured Exceptions.

### CODE REVIEW

File: src/main/java/com/github/catalystcode/fortis/speechtotext/lifecycle/MessageReceiver.java [hunk 1]
```diff
@@ -14,35 +14,46 @@
...
+            log.debug("Got message at path " + path + " with payload '" + body + "'");
+
+            if (SPEECH_HYPOTHESIS.equalsIgnoreCase(path)) {
+                SpeechHypothesisMessage.handle(body, onHypothesis);
+            } else if (SPEECH_PHRASE.equalsIgnoreCase(path)) {
+                SpeechPhraseMessage.handle(body, onResult);
+            } else if (TURN_END.equalsIgnoreCase(path)) {
+                TurnEndMessage.handle(sender, endLatch);
+            }
+        } catch (Exception e) {
```
Comment (imkrmgn) [inline src/main/java/com/github/catalystcode/fortis/speechtotext/lifecycle/MessageReceiver.java]: It's easier to catch in `NvMessageReceiver#onTextMessage` than to catch here. And there is little modiftion.

**Review by imkrmgn — CHANGES_REQUESTED** [review]
Hi @drummer3333 .

I commented because I was concerned about your change.
I am happy to think about my opinion.

**Comment by c-w** [comment]
@drummer3333 Thanks for your pull request! Much appreciated. Could you please take a look at the comment of @imkrmgn and also update your branch with the latest changes from master? Thanks!

**Comment by c-w** [comment]
There's now an official [Java/Android SDK](https://docs.microsoft.com/en-us/azure/cognitive-services/speech/getstarted/getstartedjavaandroid) so I'm archiving this repository and closing the pull request. Please migrate to the official SDK. Thanks!

**Pull request closed without merging**